man/man8/zfs.8

```diff
@@ -3829,6 +3829,31 @@ parameters with the
 .Fl o
 options.
 .Pp
+The added security provided by raw sends adds some restrictions to the send
+and receive process. ZFS will not allow a mix of raw sends and non-raw sends.
+Specifically, any raw incremental sends that are attempted after a non-raw
+send is received will fail. Non-raw sends do not have this restriction and,
```

Comment:
I would replace "send" with "receive" above.  It's the receive that will fail; the sending side doesn't know/care.